### DISCUSSION TITLE
Make "top discussions this past month" public

### DISCUSSION THREAD

**Discussion by marc-costea** [community/community#61613] [New to GitHub]
Hey guys,

can you please give me a hint, how to make the section "Top discussion this past month" viewable for public?
We've created a new organization account, and if I'm logged in I can see this section, but when not beeing loggend in, it's not visible and I have no idea where to make it public?

![image](https://github.com/community/community/assets/128606937/5a0b646a-549c-4c47-b0e2-168e4e94e8a3)

PS: Can you also tell me how this section is called?
These "upper right corner" boxes, cause I'm also looking to add more there (like "top language").

Regards,
Marc

**Comment by Kaparrs** [comment]
You might need a lot of discussions for that to appear 🤷‍♂️

**Comment by marc-costea** [comment]
@JajarGaming: Got any paper or page with some official info about this?
maybe an official amount when it switches from private to public?

What about my other question with the "top language"?
Do you know how you call this information boxes in general?

**Reply by Kaparrs** [reply]
Not really... I work in private organisations, so not sure
